common/lib/capa/capa/tests/test_targeted_feedback.py

```diff
@@ -100,7 +100,7 @@ def test_targeted_feedback_student_answer1(self):
         problem.student_answers = {'1_2_1': 'choice_3'}
 
         the_html = problem.get_html()
-        without_new_lines = the_html.replace("\n", "")
+        without_new_lines = the_html.replace("\\n", "").replace("\n", "")
```

Comment:
@feanil there is difference coming in the outputs. I have fixed in tests. But do we need to do some debugging about `\\n\\n` in python3.
Python2
```
u'<div><p>What is the correct answer?</p><div class="wrapper-problem-response" tabindex="-1" aria-label="Question 1" role="group"><div>{\'status\': Status(\'unsubmitted\'), \'input_type\': \'radio\', \'STATIC_URL\': \'/dummy-static/\', \'choices\': [(\'choice_0\', u\'wrong-1\'), (\'choice_1\', u\'wrong-2\'), (\'choice_2\', u\'correct-1\'), (\'choice_3\', u\'wrong-3\')], \'submitted_message\': \'Answer received.\', \'value\': \'choice_2\', \'msg\': \'\', \'show_correctness\': \'always\', \'name_array_suffix\': \'\', \'id\': \'1_2_1\', \'describedby_html\': Markup(u\'aria-describedby="status_1_2_1"\'), \'response_data\': {\'descriptions\': OrderedDict(), \'label\': \'\'}}</div></div><targetedfeedbackset><section class="targeted-feedback-span"><span><targetedfeedback explanation-id="feedbackC" role="group" aria-describedby="1_2_1-legend"><span class="sr">Correct</span><div class="detailed-targeted-feedback-correct"><p>Targeted Feedback</p><p>Feedback on your correct solution...</p></div></targetedfeedback></span></section></targetedfeedbackset><div>{\'id\': \'1_solution_1\'}</div></div>'
```

Python3
```
'<div>\n    <p>What is the correct answer?</p>\n    <div class="wrapper-problem-response" tabindex="-1" aria-label="Question 1" role="group"><div>{\'choices\': [(\'choice_0\', \'wrong-1\'), (\'choice_1\', \'wrong-2\'), (\'choice_2\', \'correct-1\'), (\'choice_3\', \'wrong-3\')], \'value\': \'choice_2\', \'input_type\': \'radio\', \'status\': Status(\'unsubmitted\'), \'response_data\': {\'descriptions\': OrderedDict(), \'label\': \'\'}, \'submitted_message\': \'Answer received.\', \'id\': \'1_2_1\', \'STATIC_URL\': \'/dummy-static/\', \'msg\': \'\', \'name_array_suffix\': \'\', \'describedby_html\': Markup(\'aria-describedby="status_1_2_1"\'), \'show_correctness\': \'always\'}</div></div>\n\n    <targetedfeedbackset>\n        <section class="targeted-feedback-span"><span>b\'<targetedfeedback explanation-id="feedbackC" role="group" aria-describedby="1_2_1-legend">\\n        <span class="sr">Correct</span><div class="detailed-targeted-feedback-correct">\\n            <p>Targeted Feedback</p>\\n            <p>Feedback on your correct solution...</p>\\n        </div>\\n        </targetedfeedback>\\n\\n    \'</span></section></targetedfeedbackset>\n\n    <div>{\'id\': \'1_solution_1\'}</div></div>'
```